profiles/bicycle.lua

```diff
@@ -338,7 +338,7 @@ function way_function (way, result)
     result.forward_mode = mode.train
     result.backward_mode = mode.train
     -- railways
-    if (not access and data.highway) or profile.access_tag_whitelist[access] then
+    if (not access and data.highway) or (access and not profile.access_tag_blacklist[access]) then
```

Comment:
:+1: but the second clause is not similar to the upper condition. Here condition is *access and not in the blacklist* and upper one is *access and in the blacklist*. it will be ok to fix it above. The original problem is that  the set {¬(whitelist ∪ blacklist)} is non-empty and edges with access tags from the set have undetermined speeds and travel modes but this was ignored before.